### PR TITLE
German translation for notification badge

### PR DESCRIPTION
**Pull request by phgross**
There is no corresponding german translation therefore we use `GEVER Benachrichtigung`.

additional:
- Replace the term `Benachrichtigungsmelder` with `Benachrichtigungsanzeige` in documentation
- Update schema .inc file in api documentation

### CODE REVIEW

File: docs/public/dev-manual/api/schemas/ftw.mail.mail.inc [hunk 1]
```diff
@@ -70,7 +70,7 @@
        :Datentyp: ``Tuple``
        
        
-       :Beschreibung: Schlagwörter zur Umschreibung eines Dossiers. Nicht zu verwechseln mit der Ordnungsposition
+       :Beschreibung: Schlagwörter zur Umschreibung eines Dokuments. Nicht zu verwechseln mit der Ordnungsposition. <br>ACHTUNG: Beachten Sie bei der Verwendung von Schlagworten die Datenschutzvorgaben (z.B. keine Eigennamen).
```
Comment (deiferni) [inline docs/public/dev-manual/api/schemas/ftw.mail.mail.inc]: this contains inline `br`s, can `rst` deal with this?

File: opengever/activity/locales/de/LC_MESSAGES/opengever.activity.po [hunk 2]
```diff
@@ -105,7 +105,7 @@ msgstr "Aktivität"
 #. Default: "Badge"
 #: ./opengever/activity/browser/settings.py
 msgid "label_badge"
-msgstr "Badge"
+msgstr "GEVER Benachrichtigung"
```
Comment (deiferni) [inline opengever/activity/locales/de/LC_MESSAGES/opengever.activity.po]: why also GEVER? shouldn't that be obvious to the user 😄 ?

File: docs/public/dev-manual/api/schemas/ftw.mail.mail.inc (hunk 1 above)
Comment (phgross) [reply docs/public/dev-manual/api/schemas/ftw.mail.mail.inc]: It displays the <br> tag as text. The <br> tag is part of the field description, I just updated documentation. This update was missing in the previous PR.

File: opengever/activity/locales/de/LC_MESSAGES/opengever.activity.po (hunk 2 above)
Comment (phgross) [reply opengever/activity/locales/de/LC_MESSAGES/opengever.activity.po]: Yes and No. GEVER stands for Application. So its clearer that the In-Application Notifications are meant.

Comment (deiferni) [reply opengever/activity/locales/de/LC_MESSAGES/opengever.activity.po]: i see, works for me 😄 .

**Pull request merged**